lib/daru/io/exporters/excel.rb

```diff
@@ -23,32 +34,42 @@ class Excel < Base
         #
         #   Daru::IO::Exporters::Excel.new(df, "dataframe_df.xls").call
         #
-        # @todo The +opts+ parameter isn't used while creating the Excel Spreadsheet
-        #   yet. Implementing this feature will greatly allow the user to generate a
-        #   Spreadsheet of their choice.
-        def initialize(dataframe, path, **options)
+        # @example Writing to an Excel file with options
+        #   df = Daru::DataFrame.new([[1,2],[3,4]], order: [:a, :b])
+        #
+        #   #=> #<Daru::DataFrame(2x2)>
+        #   #      a   b
+        #   #  0   1   3
+        #   #  1   2   4
+        #
+        #   normal  = { color: :blue }
+        #   heading = { color: :red, weight: :bold }
+        #
+        #   Daru::IO::Exporters::Excel.new(df,
+        #     "dataframe_df.xls",
+        #     data_options: normal,
+        #     header_options: heading
+        #   ).call
```

Comment:
I'm totally not sure, but don't you think it is something else? Like "format of ...":
```ruby
Daru::IO::Exporters::Excel.new(formatting: {header: blah, data: blah}).call(path)
# or even...
Daru::IO::Exporters::Excel.new(formatting: {order: blah, index: blah, data: blah}).call(path)
```
(BTW, as you can see by my example, I am totally not sure where `path` should be passed... We can discuss it later, though, leave your version)